### ISSUE TITLE
improve handling of url() in max-line-length

### ISSUE THREAD

**Issue by onlywei** [stylelint/stylelint#605]
With the settings:

```json
{
  "rules": {
    "declaration-colon-space-after": "always-single-line",
    "max-line-length": 100
  }
}
```

And the code:

```css
.foo {
  background-image:
      url('url-that-is-more-than-100-chars-long');
}
```

The `declaration-colon-space-after` will throw a warning here.

**Comment by davidtheclark** [comment]
I think the bug here is specific to `declaration-colon-space-after`: it shouldn't reject that.

**Comment by davidtheclark** [comment]
Actually changed my mind. I think declaration-colon-space-after is right in applying the single-line rule whenever the value itself is only a single-line.

What is needed is better handling of urls in max-line-length. I have an idea going here ...

**Comment by jeddy3** [comment]
@onlywei FYI, released as `3.2.1`

**Comment by onlywei** [comment]
@jeddy3 @davidtheclark thanks!! That was so fast!! 

Now the Atom `linter-stylelint` just needs a bump :+1:
